src/main/java/org/opensrp/web/rest/ProductCatalogueResource.java

```diff
@@ -48,7 +49,8 @@
 	public List<ProductCatalogue> getAll(
```

Comment:
For backward compatibility we need to keep the old method (overload getAll) since it is Public and not deprecated.

Once implemented the deprecated method can invoke the new method but pass null for the new parameters

Reply:
The legacy `getAll` method can be marked as `@Deprecated`